drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -222,9 +242,13 @@ function DragNDropTemplates(url_name) {
         if (ctx.popup_html && !ctx.last_action_correct) {
             popupSelector += '.popup-incorrect';
         }
+        // Render only items_in_bank and items_placed_unaligned here;
+        // items placed in aligned zones will be rendered by zoneTemplate.
         var is_item_placed = function(i) { return i.is_placed; };
         var items_placed = $.grep(ctx.items, is_item_placed);
         var items_in_bank = $.grep(ctx.items, is_item_placed, true);
+        var is_item_placed_unaligned = function(i) { return i.is_placed && !i.zone_align; };
```

Comment:
@pomegranited Since you're using `is_item_placed_unaligned` to filter `items_placed` in the line below, I think you can remove `i.is_placed` from the body of the function.